astro/exec2/process.go

```diff
@@ -106,18 +107,65 @@ func (p *Process) Run() error {
 		p.config.ExpectedSuccessCodes = []int{0}
 	}
 
+	var resultError error
+	var resultTime time.Duration
+
 	// Run the process
 	started := time.Now()
-	err := p.execCmd.Run()
+	if err := p.execCmd.Start(); err != nil {
+		logger.Error.Print(err) // Command not found on PATH, not executable, etc.
```

Comment:
Let's get rid of all `logger.Error` statements. If there are errors, we should bubble them up to the caller to decide what to do. Anything other than that is for tracing/debugging.

Reply:
I've changed this to accumulate errors using multierror